roles/common/tasks/ufw_rules.yml

```diff
@@ -0,0 +1,29 @@
+---
+# set app specific ufw rules 
+- name: Allow all access to required ports
+  community.general.ufw:
+    rule: allow
+    port: "{{ item }}"
+    proto: tcp
+  loop:
+    - 22
+    - 80
+    - 443
+    - 7077
+    - 8080
+    - 8081
+    - 8480
```

Comment:
This should be public. This bypasses the nginx proxy pass and htaccess restrictions